app/Http/Controllers/PagesController.php

```diff
@@ -63,4 +63,33 @@ public function help()
     {
         return view('help');
     }
+
+    /**
+     * @return comments.blade.php
+     */
+    public function comments()
+    {
+        $projects = Project::orderBy('created_at', 'desc')->paginate(12);
+
+        return view('pages.comments', ['projects' => $projects]);
+    }
+
+    /**
+     * @return views.blade.php
+     */
+    public function views()
+    {
+       $projects = Project::orderBy('views', 'desc')->paginate(12);
+
+        return view('pages.views', ['projects' => $projects]);
+    }
+    /**
+     * @return likes.blade.php
+     */
+    public function likes()
+    {
+        $projects = Project::orderBy('likes', 'desc')->paginate(12);
+
+        return view('pages.likes', ['projects' => $projects]);
```

Comment:
You don't need to have different views, use a query string to determine the kind of content that shows on the landing page. 

For instance, `pibbble-staging.herokuapp.com/submissions?sort=views` for views and `pibbble-staging.herokuapp.com/submissions?sort=likes` and `pibbble-staging.herokuapp.com/submissions?sort=comments` is a very good way of doing it.

So when the user clicks on this links, it performs those queries based on these query strings and displays them on the normal page, thus eliminating the need for a different page for `likes` and `views`